db/schema.rb

```diff
@@ -217,6 +217,10 @@
     t.datetime "requested_at"
     t.boolean "ial2_strict"
     t.string "issuer"
+    t.datetime "verify_doc_submitted_at"
```

Comment:
Since these are used for more than just docs, could the columns be renamed to drop the `doc` part?

Reply:
we would need to make it clear that we are verifying a document as opposed to pii or something else.  we also use document references in proofing components.

Reply:
ah ok, sounds good!